### PR TITLE
feat(sampling): Add parent projects alert [TET-338]

### PR DESCRIPTION
**Pull request by matejminar**
image

Depends on https://github.com/getsentry/sentry/pull/39539

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

**Comment by andriisoldatenko** [comment]
BE PR https://github.com/getsentry/sentry/pull/39546, please wait for merging it.

File: static/app/views/settings/project/server-side-sampling/samplingFromOtherProject.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import {ServerSideSamplingStore} from 'sentry/stores/serverSideSamplingStore';
+
+import {SamplingFromOtherProject} from './samplingFromOtherProject';
+import {getMockData, mockedSamplingDistribution} from './testUtils';
+
+export const samplingBreakdownTitle = 'Transaction Breakdown';
+
+describe('Server-Side Sampling - SamplingFromOtherProject', function () {
+  afterEach(function () {
+    act(() => ProjectsStore.reset());
```
Comment (priscilawebdev) [inline static/app/views/settings/project/server-side-sampling/samplingFromOtherProject.spec.tsx:13]: do we need `act` here?

Comment (matejminar) [reply static/app/views/settings/project/server-side-sampling/samplingFromOtherProject.spec.tsx:13]: it was failing without it and I've seen other tests fighting that with an act

Comment (priscilawebdev) [reply static/app/views/settings/project/server-side-sampling/samplingFromOtherProject.spec.tsx:13]: if you use 

` beforeEach(function () {
    ProjectsStore.reset();
    ServerSideSamplingStore.reset();
  });
` 

it seems to solve the issue. ANother alternative would be to have this reset functions inside of each test

File: static/app/types/sampling.tsx [hunk 2]
```diff
@@ -102,6 +102,13 @@ export type SamplingRule = {
 
 export type SamplingDistribution = {
   endTimestamp: string | null;
+  parentProjectBreakdown:
+    | null
+    | {
+        percentage: number;
+        project: string;
+        project_id: number;
```
Comment (andriisoldatenko) [inline static/app/types/sampling.tsx:110]:
```suggestion
        projectId: number;
```

**Pull request merged**